bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -2,147 +2,121 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% add_lang_files "firefox/new" %}
+{% add_lang_files "firefox/new/horizon" %}
```

Comment:
Yes, at this point it's better to keep the existing lang file name even if the template changes.